apps/nsqlookupd/main.go

```diff
@@ -0,0 +1,22 @@
+// +build !windows
+
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+)
+
+func main() {
+	daemon := start()
```

Comment:
sorry, did not realize this was for `nsqlookupd`, in which case it should be `nsqlookupd` :)